src/Components/WebAssembly/Server/src/Microsoft.AspNetCore.Components.WebAssembly.Server.csproj

```diff
@@ -12,6 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore.Components.Endpoints" />
```

Comment:
It might not make any difference, and there's probably an important reason you did it this way, but TBH I was expecting the dependency be Endpoints->WebAssembly.Server rather than this way around.

That's basically because WebAssembly.Server is an existing, self-contained thing which doesn't need anything from Endpoints to do its existing job, whereas Endpoints is a kind of aggregation of our functionality from many areas so it would be intuitive for it to pull in WebAssembly.Server to achieve the new functionality.

If it is possible to swap this around and have `AddWebAssemblyComponents` live inside `Endpoints`, that would seem more obvious. But if that's impractical or undesirable for some reason I'd be interested to understand!

Reply:
Having the dependency be Endpoints->WebAssembly.Server means we're pulling WebAssembly.Server into the shared framework. I wasn't sure if there was a specific reason why WebAssembly.Server wasn't already in the shared framework, but I don't see a reason why it shouldn't be included.

I've now swapped the direction of this dependency and added WebAssembly.Server into the shared framework, but I'm happy to revert it back if there's a reason we don't want to make that change 🙂

Reply:
> added WebAssembly.Server into the shared framework

That sounds good to me. I don't think there was any particular benefit in making people reference that package explicitly in the past.

Reply:
We discussed this separately and found we couldn't really invert the dependency here because that would pull other transitive dependencies into the shared framework. Marking this discussion as resolved. We may revisit this in the future.